app/src/components/UI/Molecules/Form Group/FormGroup.tsx

```diff
@@ -1,30 +1,38 @@
 import React from "react";
 import { Container, Form, Icon } from "semantic-ui-react";
 
-const FormGroup: React.FC<{ disabled: boolean }> = ({ disabled }) => {
+const FormGroup: React.FC<{ disabled: boolean; formLabels: string[] }> = ({
```

Comment:
It looks nice. I'm not sure how it would work with event handlers. Could you add the following props or something similar so that I can be sure that it can be integrated into `AccountProfile`?

```typescript
interface Props {
  value: UserProtected,
  onChange: (value: UserProtected) => void,
}
```

Reply:
I added the two props. There are compilation errors in AccountProfile, as expected.